docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md

```diff
@@ -110,8 +110,8 @@ For the Fusion Middleware Infrastructure image, you must install the [required p
 
 
   ```bash
-    $ docker pull container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.3-200109
-    $ docker tag container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.3-200109  oracle/fmw-infrastructure:12.2.1.3
+    $ docker pull container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.3-200316
```

Comment:
I am wondering if the  soa-suite doc also needs to be updated to use fmw-infrastructure:12.2.1.4 image now.

Reply:
I don't know. @rjeberhard , any thoughts?

Reply:
No, there is no soa-suite:12.2.1.4 image yet.